spyder/plugins/variableexplorer/plugin.py

```diff
@@ -39,12 +39,14 @@ class VariableExplorer(SpyderDockablePlugin, ShellConnectMixin):
     def get_name():
         return _('Variable explorer')
 
-    def get_description(self):
-        return _('Display, explore load and save variables in the current '
-                 'namespace.')
+    @staticmethod
+    def get_description():
+        return _("Display, explore, load and save global variables generated "
+                 "after execution")
```

Comment:
```suggestion
        return _("Explore, edit, save and load the lists, arrays, dataframes "
                 "and other global variables generated by running your code.")
```

* Explore is a superset of display in this context, and is the task the user performs (like the others) with the plugin, like the others
* Also important that the Variable Explorer allows editing many data types
* Load chronologically comes after save
* Mention a few of the most popular supported datatypes as well as giving the user a reference point for what is being shown
* More clearly and conventionally describe when this is shown in a more user-centric way ("generated by your code", and avoid grammar/syntactic ambiguity with "generated after execution", which implies the _generation_ happens after execution rather than the display is after execution.